scene/animation/animation_player.h

```diff
@@ -184,9 +185,20 @@ class AnimationPlayer : public Node {
 		StringName next;
 		Vector<TrackNodeCache *> node_cache;
 		Ref<Animation> animation;
+		StringName animation_library;
+		uint64_t last_update = 0;
 	};
 
 	Map<StringName, AnimationData> animation_set;
+
+	struct AnimationLibraryData {
+		StringName name;
+		Ref<AnimationLibrary> library;
+		bool operator<(const AnimationLibraryData &p_data) const { return name.operator String() < p_data.name; }
```

Comment:
Is this a `String < StringName` comparison? Shouldn't it be `String < String` comparison?

Reply:
Good catch, I have no idea what it is comparing. It appears to work because it sorts the libraries properly, but it does not look super safe.